data/transportation/roads-system/index.html

```diff
@@ -159,9 +161,7 @@ <h5 id="RoadCenterlines-updates">Updates</h5>
         <li><a href="{% link _posts/2022-11-14-utah-sgid-statewide-roads-data-layer-updates-11142022.md %}">November 14,
             2022</a>                 
         <li><a href="{% link _posts/2022-10-07-utah-sgid-statewide-roads-data-layer-updates-10072022.md %}">October 07,
-            2022</a>          
-        <li><a href="{% link _posts/2022-09-07-utah-sgid-statewide-roads-data-layer-updates-09072022.md %}">September 07,
```

Comment:
**thought**: we should start deleting these old posts as they are no longer relevant

Reply:
yup, I agree.  I figure i'll leave one year, plus one month.  so, good catch.  i can remove sept.

Reply:
oh, i see what you're saying.  delete the file.  right?

Reply:
good thought.